components/raftstore-v2/src/worker/pd/store.rs

```diff
@@ -14,13 +14,23 @@ use pd_client::{
     PdClient,
 };
 use prometheus::local::LocalHistogram;
-use slog::{error, warn};
-use tikv_util::{metrics::RecordPairVec, store::QueryStats, time::UnixSecs, topn::TopN};
+use raftstore::store::{metrics::STORE_SNAPSHOT_TRAFFIC_GAUGE_VEC, util::LatencyInspector};
+use slog::{error, info, warn};
+use tikv_util::{
+    metrics::RecordPairVec,
+    store::QueryStats,
+    time::{Instant as TiInstant, UnixSecs},
+    topn::TopN,
+};
 
 use super::Runner;
+use crate::router::StoreMsg;
 
 const HOTSPOT_REPORT_CAPACITY: usize = 1000;
 
+/// Max limitation of delayed store_heartbeat.
```

Comment:
Put a time unit or a value explaination (5min) in the comment would be nice.

Reply:
Thx.